stubs/keyboard/keyboard/mouse.pyi

```diff
@@ -42,7 +43,7 @@ def click(button: _MouseButton = "left") -> None: ...
 def double_click(button: _MouseButton = "left") -> None: ...
 def right_click() -> None: ...
 def wheel(delta: int = 1) -> None: ...
-def move(x: SupportsInt, y: SupportsInt, absolute: bool = True, duration: float = 0) -> None: ...
+def move(x: ConvertibleToInt, y: ConvertibleToInt, absolute: bool = True, duration: float = 0) -> None: ...
```

Comment:
Same question here: what's the reason why a user might want to pass a `str` to this function?

Reply:
Unless some API give screen coordinates or relative positions in `str`. I'm not aware of any screen/window API that do that.